lib/action_view/component/test_helpers.rb

```diff
@@ -3,8 +3,8 @@
 module ActionView
   module Component
     module TestHelpers
-      def render_component(component, &block)
-        Nokogiri::HTML(component.render_in(ApplicationController.new.view_context, &block))
+      def render_inline(component, **args, &block)
```

Comment:
Given that you seem to care about not breaking the 1.0 API until v2.0, I think it's worth noting that renaming `ActionView::Component` to `ActionView::Component::Base` in v1.1.0 was also a breaking change.

I don't think it's particularly difficult for people to fix when they upgrade, but figured I'd point it out.

Reply:
Just some thoughts given the above:

I think you could either treat 1.x as still beta software that can have breaking changes in minor upgrades (the way many packages are in the 0.x stage), or just go ahead and jump to 2.0 now since it's still early in the adoption of this gem and you already have identified a few breaking API changes you want to make. (If you choose the former, documenting it in the README would probably be a good idea.)